compute/instance-templates/create-instance-templates/create_instance_templates_test.go

```diff
@@ -207,4 +207,40 @@ func TestCreateInstanceTemplatesSnippets(t *testing.T) {
 	if err = op.Wait(ctx); err != nil {
 		t.Errorf("unable to wait for the operation: %v", err)
 	}
+
+	t.Run("regional template", func(t *testing.T) {
```

Comment:
issue: break these into individual tests methods. We should ideally have one test method for each sample.

Reply:
Thank you!